packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -669,15 +685,47 @@ export default class TabAdvanced extends Component {
     return isPciPassthroughAvailable(host) ? undefined : _('onlyAvailableXcp8.3OrHigher')
   }
 
+  _confirmUefiMode = async () => {
+    const { vm, vmSecurebootReadiness } = this.props
+    const confirmNeeded =
+      vmSecurebootReadiness === 'disabled' ||
+      vmSecurebootReadiness === 'ready' ||
+      vmSecurebootReadiness === 'ready_no_dbx'
+
+    if (confirmNeeded) {
+      SECUREBOOT_STATUS_MESSAGES[vmSecurebootReadiness]
+
+      await confirm({
+        title: _('propagateCertificatesTitle'),
+        body: <p>{_('propagateCertificatesConfirm')}</p>,
+      })
+    }
+
+    await vmSetUefiMode(vm, 'user')
```

Comment:
This fails with `method not found: vm.setUefiMode`.

In addition to this, the error popup has a "show logs" button, but when clicked it doesn't show any log that is related to the error.